test/integration/data-types.test.js

```diff
@@ -460,5 +460,26 @@ describe(Support.getTestDialectTeaser('DataTypes'), function () {
         expect(record.stamp).to.be.eql(testDate);
       });
   });
+  
+  if (dialect === 'mssql') {
+    it('should parse ROWVERSION field', function () {
+      const Model = this.sequelize.define('model', {
+        name: Sequelize.STRING,
+        version: Sequelize.ROWVERSION
+      });
+
+      const sampleData = {
+        name: 'test'
+      };
 
+      return Model.sync({ force: true }).then(() => {
+        return Model.create(sampleData);
+      }).then(() => {
+        return Model.findAll().get(0);
+      }).then(model => {
+        expect(model.get('version')).to.be.ok;
+        expect(typeof model.get('version')).to.be.eql('object');
```

Comment:
It's not really a number in MSSQL. It's an object type. If I don't remember wrongly, the value is in Binary.

>It is not possible to convert the TimeStamp colunm into a Date & Time format. Microsoft has renamed the TimeStamp data type to RowVersion to avoid the confusion.

> The timestamp/rowversion data type is really just a Database Transaction ID, and it can also be stored in a column of the Binary(8) data type.

[Source, albeit not a very credible one](https://dba.stackexchange.com/questions/98312/how-to-convert-timestamp-rowversion-to-a-date-and-time-in-sql-server)